lib/kwtool_web/templates/layout/app.html.eex

```diff
@@ -4,29 +4,26 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <title>Kwtool · Phoenix Framework</title>
+    <title>KWTool - Keyword research tool</title>
     <link rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
     <script defer type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
   </head>
   <body>
```

Comment:
In order to target screens, a body class must be added and be based on the controller name and action e.g. given the controller `page_controller.ex` and the action `index` then the body class should be: `page index`. Then it can be target in the screen stylesheet as:

```css
body.page.index {

}
```

You can check how Micky achieved this in his IC 👉 https://github.com/hanam1ni/google-crawler/blob/development/lib/google_crawler_web/templates/layout/app.html.eex#L11

Reply:
I added the code for generating the body class.